firestore/cloud-client/snippets.py

```diff
@@ -1,4 +1,5 @@
-# Copyright 2017, Google, Inc.
+# Copyright 2019 Google, LLC.
```

Comment:
Please preserve the original copyright year

Reply:
Haha, I was focusing on the `Inc.` vs `LLC` and didn't even notice it changed to 2019 😆